apps/aeutils/src/aeu_requests.erl

```diff
@@ -94,6 +96,22 @@ send_tx(Peer, SignedTx) ->
             Error
     end.
 
+%% NOTE that this is part of the internal API, thus the standard peer
+%% uris will not work
+new_spend_tx(IntPeer, #{sender_pubkey := Ks,
```

Comment:
who is calling this function? Is aeu_requests.erl expected to gather more transaction specific code? Maybe we should contain it in transaction specific module?

Reply:
It could well be placed elsewhere, but does reuse the support code in this module.
I assume it will primarily be used for testing purposes.